block-ship/src/types.rs

```diff
@@ -67,7 +55,7 @@ impl BlockWrapper {
     }
 
     pub fn to_chunks(&self) -> Result<Vec<TransmissionMessage>> {
-        let cid_marker = &self.cid[..4];
+        let cid_marker = &self.cid[..10];
```

Comment:
Magic number, consider making an expressive variable or add a comment on the function.